esphome/components/mopeka_pro_check/sensor.py

```diff
@@ -89,6 +97,26 @@ def small_distance(value):
                 device_class=DEVICE_CLASS_BATTERY,
                 state_class=STATE_CLASS_MEASUREMENT,
             ),
+            cv.Optional(CONF_SIGNAL_STRENGTH): sensor.sensor_schema(
+                unit_of_measurement=UNIT_EMPTY,
+                icon=ICON_SIGNAL,
+                accuracy_decimals=0,
+                device_class=DEVICE_CLASS_SIGNAL_STRENGTH,
+                state_class=STATE_CLASS_MEASUREMENT,
+            ),
+            cv.Optional(CONF_IGNORED_READS): sensor.sensor_schema(
+                unit_of_measurement=UNIT_EMPTY,
+                icon=ICON_COUNTER,
+                accuracy_decimals=0,
+                entity_category=ENTITY_CATEGORY_DIAGNOSTIC,
+            ),
+            # 0 == None
+            # 1 == LOW
+            # 2 == MED
+            # 3 = HIGH
+            cv.Optional(CONF_MINIMUM_SIGNAL_QUALITY_TO_REPORT, default=2): cv.int_range(
+                min=0, max=3
+            ),
```

Comment:
Probably doesn't need `to_report`
Also should be an enum
```suggestion
            cv.Optional(CONF_MINIMUM_SIGNAL_QUALITY, default="MEDIUM"): cv.enum(SIGNAL_QUALITIES, upper=True),
```